airbyte-ci/connectors/metadata_service/lib/metadata_service/gcs_upload.py

```diff
@@ -119,6 +138,29 @@ def _icon_upload(metadata: ConnectorMetadataDefinitionV0, bucket: storage.bucket
         return False, f"No Icon found at {local_icon_path}"
     return upload_file_if_changed(local_icon_path, bucket, latest_icon_path)
 
+def _docs_upload(metadata: ConnectorMetadataDefinitionV0, bucket: storage.bucket.Bucket, metadata_file_path: Path, latest: bool) -> Tuple[bool, str]:
+    connector_type = metadata.data.connectorType
+    doc_file_name = metadata.data.documentationUrl.split('/')[-1]
+    docs_folder_path = metadata_file_path.parents[3] / DOCS_FOLDER_PATH / f"{connector_type}s"
+    
+    local_doc_path = docs_folder_path / f"{doc_file_name}.md"
+    local_inapp_doc_path = docs_folder_path / f"{doc_file_name}.inapp.md"
```

Comment:
💅  it seems like inapp docs are distinct. 

We should either
1. Move the variables for inapp next to where they are used.
2. Split the inapp upload and local upload into a "private" functions e.g. `_inapp_doc_upload`

I believe I prefer 2

Reply:
I went ahead and just modified this function to take in one more `inapp: bool` parameter, to make it work for either one